src/types/jobs/schema.graphql

```diff
@@ -0,0 +1,19 @@
+type RelatedJobs {
+    id: String,
```

Comment:
The type structures are good, but it's a good practice to mark the most important properties as required. For example, let's say the `id` is the most important property from the `RelatedJobs` type, then you should mark it as required by just adding an exclamation mark after the data type:

```graphql
type RelatedJobs {
  id: String!,
  ...
}
```